pyproject.toml

```diff
@@ -91,7 +91,6 @@ docs = [
 ]
 notebook-dependencies = [
     "circuit-knitting-toolbox[cplex,pyscf]",
-    "quantum-serverless>=0.0.7",
```

Comment:
I'd like to go ahead and remove both serverless tutorials and all dependency on that package. As Middleware definitions evolve, I see less of a need to feature serverless in the tutorials in this package